app/src/main/java/ca/chronofit/chrono/circuit/CircuitDashboardFrag.kt

```diff
@@ -151,11 +171,11 @@ class CircuitDashboardFrag : Fragment() {
         }
 
         modalSheetView.edit_layout.setOnClickListener {
-            Toast.makeText(
-                requireContext(),
-                "\uD83D\uDEE0\uFE0F Edit circuit coming soon!! \uD83D\uDEE0\uFE0F",
-                Toast.LENGTH_SHORT
-            ).show()
+            val intent = Intent(requireContext(), CircuitCreateActivity::class.java)
+            intent.putExtra("isEdit", true)
+            intent.putExtra("circuitPosition", position)
+            dialog.dismiss()
+            startActivityForResult(intent, 10003)
```

Comment:
10003 is being used in multiple places as well.

Reply:
Yeah that's starting the activity for code 10003, the other comment you made is where I'm getting the same activity's result. Yeah I'll quickly make some constants